### PR TITLE
system-databases: Refactor/Improve clarity

### LINKED ISSUE #2642: System Databases section of manual is confusing
## Change Request

**This repository is ONLY used to solve issues related to DOCS.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

See: https://pingcap.com/docs/stable/system-tables/system-table-cluster-info/

The structure has the following as system databases (as formatted code):

```
mysql
information_schema
sql-diagnosis
```

But if I use `mysql`, the system databases are:

```
mysql> show databases;
+--------------------+
| Database           |
+--------------------+
| INFORMATION_SCHEMA |
| METRICS_SCHEMA     |
| PERFORMANCE_SCHEMA |
| mysql              |
| test               |
+--------------------+
5 rows in set (0.00 sec)
```

2. Describe your suggestion or addition.

These show really correspond to where they are in the product. For example, some of the ones under sql-diagnosis are actually information_schema tables.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by nullnotnil**
### What is changed, added or deleted? (Required)

Fixes #2642

There are actually quite a few issues with the current system tables documentation:

- It organizes them into categories that don't match where they are in MySQL (#2642)
- These are all called "system tables", but they have unique namespaces. It makes it confusing because it's quite possible that there could be a table of the same name in mysql.* and information_schema.* in future. Information_schema and metrics_schema would more correctly be called "system views", but I've not changed the navigation for now, as it was recently renamed from "System Databases". Information_schema also has meaning because it is an SQL standard. "system tables" is not.
- Some information_schema tables have their own docs page. Others do not. I went with converting everything to one page per table for consistency. This makes it similar to the SQL Statements section, which is easy to follow. 
- Several of the example outputs have not been updated for recent TiDB versions.
- Several of the example outputs do not match the commands (i.e. command= DESC, output = show create table)
- In the main information schema page: https://docs.pingcap.com/tidb/v4.0/system-table-information-schema -- several of the example tables are repeated. See for example "SLOW_QUERY table" and "ANALYZE_STATUS table" -- but there are several others such as the tikv_* and tidb_* tables (!!)
- The heading indent level was inconsistent on the main i_s page.
- The "TiDB system tables" page makes it hard to follow that this is the MySQL schema. It should say it contains both stuff for MySQL and TiDB

I have also removed the "category" metadata of these files, since this is being deprecated.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

Fixes #2642

### CODE REVIEW

File: information-schema.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+---
+
+# Information Schema
+
+Information Schema provides an ANSI-standard way of viewing system metadata. TiDB also provides a number of custom `INFORMATION_SCHEMA` tables, in addition to the tables included for MySQL compatibility.
+
+Many `INFORMATION_SCHEMA` tables have a corresponding `SHOW` command. The benefit of querying `INFORMATION_SCHEMA` is that it is possible to join between tables.
+
+| Table Name                                                                              | TiDB Extension | Description |
+|-----------------------------------------------------------------------------------------|----------------|-------------|
```
Comment (zz-jason) [inline information-schema.md:15]: how about grouping all the tidb extension tables into one section to help users get a full view of these tables.

Comment (nullnotnil) [reply information-schema.md]: Works for me. I've made this change.

File: .github/assign-by-files.yml [hunk 2]
```diff
@@ -83,7 +83,7 @@
 #   - TomShawn
 #   - 
 
-# 'system-tables/**/*.md':
+# 'information-schema/**/*.md':
 #   - TomShawn
 #   -
```
Comment (yikeke) [inline .github/assign-by-files.yml:88]:
```suggestion
'information-schema/**/*.md':
  - TomShawn
  - nullnotnil
```
Do you wanna review `information-schema` related PRs in the future? @nullnotnil

File: mysql-schema.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 ---
-title: TiDB System Tables
-summary: Learn the TiDB system tables.
-aliases: ['/docs/dev/system-tables/system-table-overview/','/docs/dev/reference/system-databases/mysql/']
+title: mysql
```
Comment (yikeke) [inline mysql-schema.md:2]:
```suggestion
title: `mysql` schema
```

File: mysql-schema.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
+summary: Learn about the TiDB system tables.
+aliases: ['/docs/dev/system-tables/system-table-overview/','/docs/dev/reference/system-databases/mysql/','/tidb/dev/system-table-overview/']
 ---
 
-# TiDB System Tables
+# mysql
```
Comment (yikeke) [inline mysql-schema.md:7]:
```suggestion
# `mysql` schema
```

File: information-schema.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
+---
+title: Information Schema
+summary: TiDB implements the ANSI-standard information_schema for viewing system metadata.
+aliases: ['/docs/dev/information-schema/information-schema-information-schema/','/docs/dev/reference/system-databases/information-schema/','/tidb/dev/system-table-information-schema/']
```
Comment (yikeke) [inline information-schema.md:4]:
```suggestion
aliases: ['/docs/dev/system-tables/system-table-information-schema/','/docs/dev/reference/system-databases/information-schema/','/tidb/dev/system-table-information-schema/']
```

File: information-schema/information-schema-metrics-tables.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
+---
+title: METRICS_TABLES
+summary: Learn the `METRICS_TABLES` system table.
+aliases: ['/docs/dev/system-tables/system-table-metrics-tables/','/docs/dev/reference/system-databases/metrics-tables/',''/tidb/dev/system-table-metrics-tables/'']
```
Comment (yikeke) [inline information-schema/information-schema-metrics-tables.md:4]:
```suggestion
aliases: ['/docs/dev/system-tables/system-table-metrics-tables/','/docs/dev/reference/system-databases/metrics-tables/','/tidb/dev/system-table-metrics-tables/']
```

**Comment by yikeke** [comment]
@zz-jason and @kissmydb said they will take a last look tomorrow. This PR is expected to be merged soon. FYI. @nullnotnil

File: information-schema/information-schema-metrics-summary.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
 title: METRICS_SUMMARY
 summary: Learn the `METRICS_SUMMARY` system table.
-aliases: ['/docs/dev/system-tables/system-table-metrics-summary/','/docs/dev/reference/system-databases/metrics-summary/']
+aliases: ['/docs/dev/system-tables/system-table-metrics-summary/','/docs/dev/reference/system-databases/metrics-summary/',''/tidb/dev/system-table-metrics-summary/'']
```
Comment (yikeke) [inline information-schema/information-schema-metrics-summary.md:4]:
```suggestion
aliases: ['/docs/dev/system-tables/system-table-metrics-summary/','/docs/dev/reference/system-databases/metrics-summary/','/tidb/dev/system-table-metrics-summary/']
```
@nullnotnil Please help fix this alias in another PR.

Comment (yikeke) [reply information-schema/information-schema-metrics-summary.md:4]: I've already fixed it in release-4.0 branch in the cherry-picked PR https://github.com/pingcap/docs/pull/3353.

Comment (yikeke) [reply information-schema/information-schema-metrics-summary.md:4]: Fixed the alias in https://github.com/pingcap/docs/pull/3367. FYI @nullnotnil @TomShawn

File: information-schema.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+| `CLUSTER_STATEMENTS_SUMMARY_HISTORY`                                                    | Provides a cluster-level view of the `CLUSTER_STATEMENTS_SUMMARY_HISTORY` table. |
+| [`CLUSTER_SYSTEMINFO`](/information-schema/information-schema-cluster-systeminfo.md)    | Provides details about kernel parameter configuration for servers in the cluster. |
+| [`DDL_JOBS`](/information-schema/information-schema-ddl-jobs.md)                        | Provides similar output to `ADMIN SHOW DDL JOBS` |
+| [`INSPECTION_RESULT`](/information-schema/information-schema-inspection-result.md)      | Triggers internal diagnostics checks. |
+| [`INSPECTION_RULES`](/information-schema/information-schema-inspection-rules.md)        | A list of internal diagnostic checks performed. |
+| [`INSPECTION_SUMMARY`](/information-schema/information-schema-inspection-summary.md)    | A summarized report of important monitoring metrics. |
+| [`METRICS_SUMMARY`](/information-schema/information-schema-metrics-summary.md)          | A summary of metrics extracted from Prometheus. |
+| `METRICS_SUMMARY_BY_LABEL`                                                              | See `METRICS_SUMAMARY` table. |
```
Comment (yikeke) [inline information-schema.md:70]: Found a typo METRICS_SUMAMARY -> METRICS_SUMMARY. Can you help fix it? @nullnotnil

Comment (yikeke) [reply information-schema.md:70]: Fixed it in https://github.com/pingcap/docs/pull/3470 (including some other fix)
